tutorials/cyber/network_mapping/README.md

```diff
@@ -0,0 +1,3 @@
+# Network Mapping Using RAPIDS
+
+To get started, please unpack the data files located in `/data/lanl` and place them in `tutorials/cyber/network_mapping/data/input/lanl`
```

Comment:
Can you include a cell in the network mapping notebook that unpacks this data for the user and copies it to the right place?

Alternatively, could you update the Dockerfile to unpack it there at image build time?

Reply:
Sure, I have updated and included a new cell